File: app/src/main/java/com/harrywoodworth/atoll/game/island/RockGrowth.java
package com.harrywoodworth.atoll.game.island;

import com.harrywoodworth.atoll.game.utility.DesignManager;

public class RockGrowth {

    public static double rock_evolution_rate = DesignManager.DEFAULT_ROCK_EVOLUTION_RATE;

    public RockGrowth(){}

    public RockGrowth(double _rock_evolution_rate) {
        rock_evolution_rate = _rock_evolution_rate;
    }

}

File: app/src/main/java/com/harrywoodworth/atoll/game/island/growths/RockGrowth.kt
package com.harrywoodworth.atoll.game.island.growths

import com.harrywoodworth.atoll.game.utility.DesignManager

class RockGrowth (
    var growth_factor: Int = DesignManager.DEFAULT_ROCK_GROWTH_FACTOR,
    var seed_count : Int = DesignManager.DEFAULT_ROCK_SEED_COUNT
) {
    fun rGrowthFactRand() : Int { return DesignManager.RANDOM_ROCK_GROWTH_FACTOR() }
    fun rSeedCountRand() : Int { return DesignManager.RANDOM_ROCK_SEED_COUNT() }
}